[2022-04-18 12-52-50] [ ERROR] [GlobalExceptionHandler:15] not-null property references a null or transient value : com.example.integradorV2.Entities.Dentist.lastName; nested exception is org.hibernate.PropertyValueException: not-null property references a null or transient value : com.example.integradorV2.Entities.Dentist.lastName
[2022-04-18 12-54-31] [ ERROR] [GlobalExceptionHandler:15] not-null property references a null or transient value : com.example.integradorV2.Entities.Dentist.lastName; nested exception is org.hibernate.PropertyValueException: not-null property references a null or transient value : com.example.integradorV2.Entities.Dentist.lastName
[2022-04-19 14-11-00] [ ERROR] [GlobalExceptionHandler:40] Error {uri='/dentists/1', errorMessage=''}
[2022-04-19 14-12-29] [ ERROR] [GlobalExceptionHandler:40] Error {uri='/dentists/1', errorMessage=''}
[2022-04-19 14-12-37] [ ERROR] [GlobalExceptionHandler:40] Error {uri='/dentists/0', errorMessage='Dentist not found'}
[2022-04-19 14-14-02] [ ERROR] [GlobalExceptionHandler:40] Error {uri='/dentists/0', errorMessage='Dentist not found'}
[2022-04-19 14-14-11] [ ERROR] [GlobalExceptionHandler:40] Error {uri='/dentists/1', errorMessage=''}
[2022-04-19 14-16-21] [ ERROR] [GlobalExceptionHandler:40] Error {uri='/dentists/1', errorMessage='ex.getMessage()'}
[2022-04-19 14-23-43] [ ERROR] [GlobalExceptionHandler:40] Error {uri='/dentists/1', errorMessage='Dentist not found'}
[2022-04-19 14-23-46] [ ERROR] [GlobalExceptionHandler:29] Error {uri='/dentists/0', errorMessage='Id can't be 0 or null'}
[2022-04-19 14-23-51] [ ERROR] [GlobalExceptionHandler:40] Error {uri='/dentists/1', errorMessage='Dentist not found'}
[2022-04-19 14-25-03] [ ERROR] [GlobalExceptionHandler:36] Error {uri='/dentists/1', errorMessage='Dentist not foundasdasd'}
[2022-04-19 14-25-09] [ ERROR] [GlobalExceptionHandler:25] Error {uri='/dentists/0', errorMessage='Id can't be 0 or null'}
[2022-04-19 14-43-30] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/dentists/0', errorMessage='Id can't be 0 or null'}
[2022-04-19 14-44-00] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/dentists/null', errorMessage='Failed to convert value of type 'java.lang.String' to required type 'java.lang.Long'; nested exception is java.lang.NumberFormatException: For input string: "null"'}
[2022-04-19 15-06-20] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/dentists/', errorMessage='not-null property references a null or transient value : com.example.integradorV2.Entities.Dentist.lastName; nested exception is org.hibernate.PropertyValueException: not-null property references a null or transient value : com.example.integradorV2.Entities.Dentist.lastName'}
[2022-04-19 15-11-34] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/appointments', errorMessage='could not execute statement; SQL [n/a]; constraint [FKQSNULA0NJ86S67OTIBU2J2HSV]; nested exception is org.hibernate.exception.ConstraintViolationException: could not execute statement'}
[2022-04-19 15-12-21] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/appointments', errorMessage='Some data of appointment like patient,dentist or dateTime is void'}
[2022-04-19 15-23-00] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/appointments', errorMessage='Id can't be 0 or null'}
[2022-04-19 15-30-59] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/appointments', errorMessage='Id can't be 0 or null'}
[2022-04-19 15-31-08] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/appointments', errorMessage='Id of Dentist can't be 0 or null'}
[2022-04-19 15-31-13] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/appointments', errorMessage='Id can't be 0 or null'}
[2022-04-19 15-31-19] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/appointments', errorMessage='Id can't be 0 or null'}
[2022-04-19 15-35-28] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/appointments', errorMessage='Patient id cannot be 0 or null'}
[2022-04-19 15-35-40] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/appointments', errorMessage='JSON parse error: Unexpected character ('}' (code 125)): expected a value; nested exception is com.fasterxml.jackson.databind.JsonMappingException: Unexpected character ('}' (code 125)): expected a value
 at [Source: (org.springframework.util.StreamUtils$NonClosingInputStream); line: 9, column: 5] (through reference chain: com.example.integradorV2.DTO.AppointmentDTO["dentist"])'}
[2022-04-19 15-37-04] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/appointments', errorMessage='JSON parse error: Unexpected character ('}' (code 125)): expected a value; nested exception is com.fasterxml.jackson.databind.JsonMappingException: Unexpected character ('}' (code 125)): expected a value
 at [Source: (org.springframework.util.StreamUtils$NonClosingInputStream); line: 9, column: 5] (through reference chain: com.example.integradorV2.DTO.AppointmentDTO["dentist"])'}
[2022-04-19 15-37-30] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/appointments', errorMessage='JSON parse error: Unexpected character ('}' (code 125)): expected a value; nested exception is com.fasterxml.jackson.databind.JsonMappingException: Unexpected character ('}' (code 125)): expected a value
 at [Source: (org.springframework.util.StreamUtils$NonClosingInputStream); line: 9, column: 5] (through reference chain: com.example.integradorV2.DTO.AppointmentDTO["dentist"])'}
[2022-04-19 15-37-53] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/appointments', errorMessage='Cannot invoke "com.example.integradorV2.DTO.DentistDTO.toString()" because "this.dentist" is null'}
[2022-04-19 15-38-34] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/appointments', errorMessage='Some data of appointment like patient,dentist or dateTime is void'}
[2022-04-19 15-39-17] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/appointments/2', errorMessage='Dentist not found'}
[2022-04-21 16-06-06] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/dentists', errorMessage='Unrecognized field "role" (class com.example.integradorV2.Entities.User), not marked as ignorable (7 known properties: "lastName", "username", "id", "email", "rol", "password", "name"])
 at [Source: UNKNOWN; byte offset: #UNKNOWN] (through reference chain: com.example.integradorV2.Entities.User["role"])'}
[2022-04-21 16-07-51] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/dentists', errorMessage='Unrecognized field "role" (class com.example.integradorV2.Entities.User), not marked as ignorable (7 known properties: "lastName", "username", "id", "email", "rol", "password", "name"])
 at [Source: UNKNOWN; byte offset: #UNKNOWN] (through reference chain: com.example.integradorV2.Entities.User["role"])'}
[2022-04-21 16-08-15] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/dentists', errorMessage='Unrecognized field "role" (class com.example.integradorV2.Entities.User), not marked as ignorable (7 known properties: "lastName", "username", "id", "email", "rol", "password", "name"])
 at [Source: UNKNOWN; byte offset: #UNKNOWN] (through reference chain: com.example.integradorV2.Entities.User["role"])'}
[2022-04-21 16-10-57] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/dentists', errorMessage='Unrecognized field "token" (class com.example.integradorV2.Entities.User), not marked as ignorable (7 known properties: "lastName", "username", "id", "email", "role", "password", "name"])
 at [Source: UNKNOWN; byte offset: #UNKNOWN] (through reference chain: com.example.integradorV2.Entities.User["token"])'}
[2022-04-21 16-12-47] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/dentists', errorMessage='Unrecognized field "token" (class com.example.integradorV2.Entities.User), not marked as ignorable (7 known properties: "lastName", "username", "id", "email", "role", "password", "name"])
 at [Source: UNKNOWN; byte offset: #UNKNOWN] (through reference chain: com.example.integradorV2.Entities.User["token"])'}
[2022-04-21 16-14-38] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/dentists', errorMessage='Unrecognized field "username" (class com.example.integradorV2.DTO.DentistDTO), not marked as ignorable (5 known properties: "lastName", "licenseNumber", "id", "appointments", "name"])
 at [Source: UNKNOWN; byte offset: #UNKNOWN] (through reference chain: com.example.integradorV2.DTO.DentistDTO["username"])'}
[2022-04-21 16-15-34] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/dentists', errorMessage='Unrecognized field "token" (class com.example.integradorV2.Entities.User), not marked as ignorable (7 known properties: "lastName", "username", "id", "email", "role", "password", "name"])
 at [Source: UNKNOWN; byte offset: #UNKNOWN] (through reference chain: com.example.integradorV2.Entities.User["token"])'}
[2022-04-21 16-17-56] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/dentists', errorMessage='Unrecognized field "token" (class com.example.integradorV2.Entities.User), not marked as ignorable (7 known properties: "lastName", "username", "id", "email", "role", "password", "name"])
 at [Source: UNKNOWN; byte offset: #UNKNOWN] (through reference chain: com.example.integradorV2.Entities.User["token"])'}
[2022-04-21 16-34-10] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/dentists', errorMessage='Unrecognized field "token" (class com.example.integradorV2.Entities.User), not marked as ignorable (7 known properties: "lastName", "username", "id", "email", "role", "password", "name"])
 at [Source: UNKNOWN; byte offset: #UNKNOWN] (through reference chain: com.example.integradorV2.Entities.User["token"])'}
[2022-04-21 16-50-59] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/dentists', errorMessage='Unrecognized field "email" (class com.example.integradorV2.DTO.DentistDTO), not marked as ignorable (8 known properties: "lastName", "appointments", "username", "licenseNumber", "id", "role", "password", "name"])
 at [Source: UNKNOWN; byte offset: #UNKNOWN] (through reference chain: com.example.integradorV2.DTO.DentistDTO["email"])'}
[2022-04-21 16-59-36] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/dentists', errorMessage='could not execute statement; SQL [n/a]; constraint ["PUBLIC.UK_R43AF9AP4EDM43MMTQ01ODDJ6_INDEX_4 ON PUBLIC.USERS(USERNAME NULLS FIRST) VALUES ( /* 1 */ 'hola' )"; SQL statement:
insert into users (id, last_name, name, password, role, username) values (default, ?, ?, ?, ?, ?) [23505-206]]; nested exception is org.hibernate.exception.ConstraintViolationException: could not execute statement'}
[2022-04-21 17-02-41] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/dentists/1', errorMessage='Dentist not found'}
[2022-04-21 17-06-46] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/dentists/21', errorMessage='Dentist not found'}
[2022-04-21 17-21-33] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/users/login', errorMessage='This User doesn't exist'}
[2022-04-21 17-53-38] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/users/login', errorMessage='No enum constant com.example.integradorV2.Entities.Role.0; nested exception is java.lang.IllegalArgumentException: No enum constant com.example.integradorV2.Entities.Role.0'}
[2022-04-21 17-54-38] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/users/login', errorMessage='Java 8 date/time type `java.time.LocalDate` not supported by default: add Module "com.fasterxml.jackson.datatype:jackson-datatype-jsr310" to enable handling (through reference chain: com.example.integradorV2.Entities.Patient["entryDate"])'}
[2022-04-23 13-14-52] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/dentists/1', errorMessage='Dentist not found'}
[2022-04-23 13-17-41] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/dentists/0', errorMessage='Dentist id cannot be 0, null or negative'}
[2022-04-23 13-17-53] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/dentists/0', errorMessage='Dentist id cannot be 0, null or negative'}
[2022-04-23 13-19-37] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/appointments', errorMessage='Request method 'PUT' not supported'}
[2022-04-23 13-19-42] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/appointments', errorMessage='Content type 'text/plain;charset=UTF-8' not supported'}
[2022-04-23 13-19-49] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/appointments', errorMessage='Patient not found'}
[2022-04-23 13-21-10] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/appointments/3', errorMessage='Appointment not found'}
[2022-04-23 13-23-09] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/patients', errorMessage='not-null property references a null or transient value : com.example.integradorV2.Entities.Patient.password; nested exception is org.hibernate.PropertyValueException: not-null property references a null or transient value : com.example.integradorV2.Entities.Patient.password'}
[2022-04-23 13-23-47] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/patients', errorMessage='not-null property references a null or transient value : com.example.integradorV2.Entities.Patient.role; nested exception is org.hibernate.PropertyValueException: not-null property references a null or transient value : com.example.integradorV2.Entities.Patient.role'}
[2022-04-23 13-31-49] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/appointments', errorMessage='Patient not found'}
[2022-04-23 13-32-02] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/appointments', errorMessage='Dentist not found'}
[2022-04-23 13-34-51] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/patients/1', errorMessage='Patient not found'}
[2022-04-23 13-35-08] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/patients', errorMessage='could not execute statement; SQL [n/a]; constraint ["PUBLIC.UK_R43AF9AP4EDM43MMTQ01ODDJ6_INDEX_4 ON PUBLIC.USERS(USERNAME NULLS FIRST) VALUES ( /* 4 */ 'patient1' )"; SQL statement:
insert into users (id, last_name, name, password, role, username) values (default, ?, ?, ?, ?, ?) [23505-206]]; nested exception is org.hibernate.exception.ConstraintViolationException: could not execute statement'}
[2022-04-23 13-37-19] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/appointments', errorMessage='Patient not found'}
[2022-04-23 13-37-25] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/appointments', errorMessage='not-null property references a null or transient value : com.example.integradorV2.Entities.Dentist.password; nested exception is org.hibernate.PropertyValueException: not-null property references a null or transient value : com.example.integradorV2.Entities.Dentist.password'}
[2022-04-23 16-44-40] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/appointments/3', errorMessage='Appointment not found'}
[2022-04-23 16-56-12] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/dentists', errorMessage='could not execute statement; SQL [n/a]; constraint ["PUBLIC.UK_R43AF9AP4EDM43MMTQ01ODDJ6_INDEX_4 ON PUBLIC.USERS(USERNAME NULLS FIRST) VALUES ( /* 4 */ 'dentist2' )"; SQL statement:
insert into users (id, last_name, name, password, role, username) values (default, ?, ?, ?, ?, ?) [23505-206]]; nested exception is org.hibernate.exception.ConstraintViolationException: could not execute statement'}
[2022-04-23 16-56-26] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/dentists', errorMessage='could not execute statement; SQL [n/a]; constraint ["PUBLIC.UK_3V8Q4EESMVOITX275Q75SVU54_INDEX_6 ON PUBLIC.DENTISTS(LICENSE_NUMBER NULLS FIRST) VALUES ( /* 4 */ 9994 )"; SQL statement:
insert into dentists (license_number, id) values (?, ?) [23505-206]]; nested exception is org.hibernate.exception.ConstraintViolationException: could not execute statement'}
[2022-04-23 17-04-55] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/appointments', errorMessage='Request method 'PUT' not supported'}
[2022-04-23 18-06-03] [ ERROR] [GlobalExceptionHandler:41] Error {uri='/appointments/1', errorMessage='Cannot invoke "com.example.integradorV2.DTO.PatientDTO.getId()" because the return value of "com.example.integradorV2.DTO.AppointmentDTO.getPatient()" is null'}
